src/components/bark/login/bark-login-page.tsx

```diff
@@ -17,11 +18,16 @@ export default async function BarkLoginPage(props: {
   }
   return (
     <>
-      <div className="flex h-screen flex-col items-center justify-center px-6">
-        <div className="w-full sm:w-[36rem]">
-          <BarkH1>{title}</BarkH1>
-          <BarkLoginForm accountType={accountType} successPath={successPath} />
-        </div>
+      <div className="flex h-screen flex-col items-center justify-center">
+        <LoginLayout accountType={accountType}>
+          <div className="mx-auto w-full max-w-[1100px] sm:w-[36rem] md:w-full">
+            <BarkH1>{title}</BarkH1>
```

Comment:
Need to update this to H2 semantically but the styling for <BarkH2/> contains a bottom border styling that is not part of the figma design

Reply:
Feel free to update BarkH2. It's not used anywhere atm.

Background, I copied the styling from this page— https://ui.shadcn.com/docs/components/typography —but have so far only really used BarkH1.

Reply:
Done 86aac9188a2aafb4f553d7e8bdcffaaf1e11b2ef